_rules/audio-text-alternative-e7aa44.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: e7aa44
-name: Audio only has a text alternative
+name: `audio` element content has text alternative
 rule_type: composite
 description: |
-  This rule checks if audio only elements have a text alternative available.
+  This rule checks if `audio` elements have a text alternative available.
```

Comment:
We should use singular, not plural. And in the previous rule,the "element" is not specified  "This rule checks `audio` is a media alternative for text on the page."

Proposal

>  This rule checks `audio` has a text alternative available

Reply:
I don't recall changing the description yesterday. It shouldn't have been changed.